### ISSUE TITLE
Интеграция с веб-браузерами

### ISSUE THREAD

**Issue by arhangelsoft** [zenden2k/image-uploader#281]
Давно сюда не заглядывал) 
Приятно что Вы снова вернулись к проекту.

Есть идея интегрировать приложение с веб-браузерами, делая доступным возможность передавать скриншоты веб-страницы. 

По простому:
Скрипт копирует картинку в буфер, а программа вытягивает её оттуда. Беда в том что из браузера нельзя просто взять и открыть программу. 
Однако ПО может поднять веб-сервер, и через AJAX запросы, общаться с браузером.

**Comment by zenden2k** [comment]
Мы это вроде уже обсуждали: https://github.com/zenden2k/image-uploader/issues/105
Я начал прикручивать веб-сервер civetweb к IU, но потом отказался от него, потому что не хватило сил и здоровья. Пока я хочу сделать хоть какой-то релиз, с исправлениями ошибок, без нового функционала, то все эти приставки beta уже надоели.

**Comment by arhangelsoft** [comment]
Да, забыл про этот тред. Попробую пока писать сторону веб-браузера. А потом уже сторона IU подтянется. С учетом native messaging вполне получится дергать IU через вставку из буфера обмена. Костыльно-топорно, но хоть так)
